thirdparty.go

```diff
@@ -19,9 +19,46 @@ const (
 	FPSUNLOCKERURL = "https://github.com/axstin/rbxfpsunlocker/releases/download/v4.4.4/rbxfpsunlocker-x64.zip"
 )
 
+var DxvkInstallMarker = filepath.Join(Dirs.Pfx, ".vinegar-dxvk")
+
+// Simple function that just tells us if the
+// 'dxvk installed' marker file exists.
+func DxvkMarkerExist() bool {
+	_, err := os.Open(DxvkInstallMarker)
+	return err == nil
+}
+
+func DxvkToggle() {
+	if Config.Dxvk {
+		DxvkInstall(false)
+
+		Config.Renderer = "D3D11"
+		Config.Env["WINEDLLOVERRIDES"] += "d3d10core,d3d11,d3d9,dxgi=n"
+	} else {
+		DxvkUninstall(false)
+	}
+}
+
 // Extract specifically DXVK's DLLs within the tarball's folders
 // to the wineprefix
-func DxvkInstall() {
+func DxvkInstall(force bool) {
+	// Flatpak provides the graphics runtime, we cannot
+	// install it ourselves as Wine will crash.
+	//   org.winehq.Wine.DLLs.dxvk
+	// We are currently using org.winehq.Wine, which provides
+	// the aformentioned runtime included. We may use our own
+	// wine later.
+	if InFlatpak {
+		// Can we install dxvk? No, we cannot.
+		panic("DXVK must be managed by the flatpak.")
```

Comment:
Just use log.Println and exit(1). panic() dumps some unnecessary debug info as well, and may not close file handles properly.

Reply:
Put off until next wine update to flatpak (TBD)